Seminars/README.md

```diff
@@ -0,0 +1,27 @@
+Welcome to our TRIC-DT Seminar Series!
+**Take part** in this community by attending the [next event](#next-events), [propose](#propose-a-topic) or upvoting a new topic or [sign up to present](#sign-up-as-a-presenter)!
+
+# 'Crafting Digital Twins' Seminar Series
+
+The “Crafting Digital Twins: Computational Methods Across Environment, Health & Infrastructure” seminar series will be an interdisciplinary platform for TRIC-DT researchers to share and discuss about the computational methods, algorithms, and models that underpin Digital Twin technology in diverse fields. This series will provide a space for exchange of knowledge and methodologies across the themes of environment, health, and infrastructure, with the aim to cultivate a unified vocabulary and uncover synergies in our work across TRIC-DT.
+
+By uniting researchers to discuss the methodological hurdles in their foundational work, this series will offer immediate value to each team member, enabling you to acquire practical insights applicable to projects at any stage, no matter how “DT-ready”. Beyond the technical learnings, presenting to a broader peer group allows us to see common challenges from fresh perspectives, potentially igniting innovative solutions to our problems.
+
+# Take Part!
+
+## Next events
+We are currently finding dates for first seminars in November & December. If you haven't received an email about this, please email sarana@turing.ac.uk for the doodle link
+
+## Sign-up as a presenter
+For existing topics, sign up as a presenter by **assigning yourself to an issue** with label `seminar series` and then **commenting with a title** and a few sentences describing your presentations. We will assign a date to each session, once we have researchers from at least two different themes signed up.
+
+We will label issues with `0/idea`, `1/in planning`, `2/scheduled` and `3/completed`.
+
+## Propose a topic
+We’ve opened an issue in this repository for each topic we’re considering. We'd like your input and "votes" on these, and your suggestions for other topics we haven't thought of. We want to invite everyone to collaborate on shaping this seminar series in something that serves the TRIC-DT research community best.
```

Comment:
Not saying you haven't thought of it @aranas - but putting the comment here, so we have it captured:

I think we need to think (a la @cassgvp) about getting people involved here. I don't think that people will magically join in voting for topics or suggesting topics without quite a bit of community work.. Ideas: Poking the researchers repeatedly in meetings, have a standing point in the biweekly team meetings, maybe do some scheduled posts in various Slack channels (interesting-events, digital-twins, tps, etc.)

Reply:
Yes very good point!

Reply:
It is something we want to revisit, since it is sort of "behind-the-scenes" community work, I will open an issue for this on our hub repository